[26.02] Docs: llama.cpp release (#11)
ROCm/llama.cpp: amd-docs/examples/llama-cpp-examples.rst

.. meta::
  :description: llama.cpp examples
  :keywords: llama.cpp, programming, C++, ROCm, example, sample, tutorial, GGML

.. _run-a-llama-cpp-example:

********************************************************************
Run a llama.cpp example
********************************************************************

The `https://github.com/ROCm/llama.cpp <https://github.com/ROCm/llama.cpp>`__ repository provides the necessary examples that exercise the functionality of your
framework.

> You can also search for llama.cpp examples on the `AMD ROCm blog <https://rocm.blogs.amd.com/>`_, 

Reviewer: ```suggestion
You can also search for llama.cpp examples in the `AMD ROCm blogs <https://rocm.blogs.amd.com/>`_, 
```

to find instructions to prepare your model and test the containers.

> Two most popular use-cases are:

Reviewer: ```suggestion
The two most popular use cases are:
```

* **llama-cli**: The main executable to run the model interactively or get a response to a prompt.
* **llama-bench**: Run a benchmark of your model with different configurations.

> Main Application: ``llama-cli`` 

Reviewer: ```suggestion
Main application: llama-cli
```

      Heart failure is a condition in which the heart cannot pump blood effectively to meet the body's needs.
      It can result from various underlying causes or contributing factors.
      The **main causes of heart failure** include:

      ---

      ### 1. **Coronary Artery Disease (CAD)**
         - Narrowing or blockage of the coronary arteries reduces blood flow to the heart muscle, weakening it over time.
         - A heart attack (myocardial infarction) can cause significant damage to the heart muscle, leading to heart failure.

      ---

      ### 2. **High Blood Pressure (Hypertension)**
         - Chronic high blood pressure forces the heart to work harder to pump blood, eventually causing the heart muscle to thicken or weaken.

      ---

      ### 3. **Cardiomyopathy**
         - Diseases of the heart muscle, such as dilated cardiomyopathy, hypertrophic cardiomyopathy, or restrictive cardiomyopathy, can impair the heart's ability to pump effectively.
      ...
      ### 11. **Other Conditions**
         - Thyroid disorders, severe anemia, or infections like myocarditis can also lead to heart failure.

      ---

      ### Prevention and Management

> Benchmark Application: ``llama-bench``

Reviewer: ```suggestion
Benchmark application: llama-bench
```

================================================================================

1. Use the CLI tool to start the application:

   .. code-block:: bash

      ./build/bin/llama-bench \
      -m /data/DeepSeek-V3-Q4_K_M/DeepSeek-V3-Q4_K_M-00001-of-00009.gguf \
      -p 16,32,64,96,128,256,512,1024,2048,4096 \
      -n 64,128,256 \
      -ngl 999

> 2. The result of the command above should be similar to the following when running on a MI300X system:

Reviewer: ```suggestion
2. The result of the command above should be similar to the following when running it on an AMD Instinct™ MI300X system:
```
